### DISCUSSION TITLE
Welcome to awesome-wm-widgets Discussions!

### DISCUSSION THREAD

**Discussion by streetturtle** [streetturtle/awesome-wm-widgets#225] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raven2cz** [comment]
Hello, 
todo-widget feature idea:  add possibility to move up/down todo row - to prioritize tasks. Thx!

**Comment by streetturtle** [comment]
Good timing) I've added it just recently, but forgot to add icons. Could you please check the latest master. It should be there and appear if there are more than two items in the list.

**Reply by raven2cz** [reply]
Works! Wonderful +1:

**Comment by rtzero-erkp** [comment]
hello, my awesome-wm's version is 4.2-4, and my os is ubuntu18. so when i use  your widgets,  error as awful.popup is not exist, i searched the dir /usr/share/awesome/lib/awful, sure popup is not exist, so what should do, pls.

**Reply by streetturtle** [reply]
I would guess that `awful.popup` didn't exist on 4.2, accoring to this [release notes](https://github.com/awesomeWM/awesome/releases/tag/v4.3):

> A new popup widget allows to bypass most of the boilerplate code and easily display widgets on the screen.

So I'd suggest to use version 4.3. In case you are struggling to build it (as I did), here is an easy way using docker: [awesome-deb-docker](https://github.com/elw00d/awesome-deb-docker)

**Comment by hosua** [comment]
Hey! 

I have a nvidia-gpu widget that I'm interested in sharing and was wondering if you're still adding widgets to this repo. 

It's currently nested in my [config repo](https://github.com/hosua/configs/tree/main/.config/awesome/hosua/hosuas-awesome-widgets/nvidia-widget) 

<img width="402" height="227" alt="image" src="https://github.com/user-attachments/assets/dced618b-562c-4013-acdb-b1e1ea8864f0" />

Thank you for the awesome (pun intended) widgets btw!

**Reply by streetturtle** [reply]
Hi! This would be amazing!!! Feel free to create a PR!

**Reply by hosua** [reply]
Done! https://github.com/streetturtle/awesome-wm-widgets/pull/488
